tests/test_2024_03_18.py

```diff
@@ -0,0 +1,286 @@
+from brownie import interface, convert, Contract, web3
+from scripts.vote_2024_03_18 import start_vote
+from utils.test.tx_tracing_helpers import *
+from utils.config import LDO_HOLDER_ADDRESS_FOR_TESTS
+from utils.test.event_validators.permission import (
+    Permission,
+    validate_permission_grant_event,
+    validate_permission_revoke_event,
+    validate_grant_role_event,
+    validate_revoke_role_event,
+)
+from utils.easy_track import create_permissions
+from utils.test.event_validators.easy_track import (
+    validate_evmscript_factory_added_event,
+    EVMScriptFactoryAdded
+)
+from utils.test.event_validators.voting import validate_change_vote_time_event, validate_change_objection_time_event
+from utils.test.event_validators.common import validate_events_chain
+from utils.test.easy_track_helpers import create_and_enact_payment_motion
+
+from scripts.vote_2024_03_18 import (
```

Comment:
Do not import these config variables - if `scripts.vote_2024_03_18.NEW_GATE_SEAL` was set incorrectly, such tests won't help
Instead, re-define these fields in this file